etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/ElementTilBrevbakerkonvertering.kt

```diff
@@ -1,45 +1,33 @@
 package no.nav.pensjon.etterlatte.maler
 
 import no.nav.pensjon.brev.template.Expression
-import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
-import no.nav.pensjon.brev.template.Language.Bokmal
-import no.nav.pensjon.brev.template.Language.English
-import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.LanguageSupport
 import no.nav.pensjon.brev.template.dsl.ListScope
 import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
 import no.nav.pensjon.brev.template.dsl.expression.equalTo
-import no.nav.pensjon.brev.template.dsl.textExpr
 import no.nav.pensjon.etterlatte.maler.ElementSelectors.children
 import no.nav.pensjon.etterlatte.maler.ElementSelectors.type
 import no.nav.pensjon.etterlatte.maler.InnerElementSelectors.children
 import no.nav.pensjon.etterlatte.maler.InnerElementSelectors.text
 import no.nav.pensjon.etterlatte.maler.InnerElementSelectors.type
 
-fun <D : BrevDTO> OutlineOnlyScope<LangBokmalNynorskEnglish, D>.konverterElementerTilBrevbakerformat(
+fun <L: LanguageSupport, D : BrevDTO> OutlineOnlyScope<L, D>.konverterElementerTilBrevbakerformat(
```

Comment:
Ikkje 100% påkravd for denne endringa, men på denne måten kan også etterlatte ha malar som ikkje nødvendigvis må fins på både nn, bm og en.